chart/templates/controller-manager.yaml

```diff
@@ -202,6 +202,7 @@ spec:
         - --leader-elect
         - --wds-name={{.Values.ControlPlaneName}}
         - --api-groups={{.Values.APIGroups}}
+        - -v=2
```

Comment:
I thought I added this in #1741 . Why was it absent now? And what about #1773 ?

Reply:
See https://github.com/kubestellar/kubestellar/pull/1771/files#r1494958505

Reply:
This file is auto generated by make chart and make local-chart.  I shouldn't have actually included it in this PR but if I run make chart this if the version of the file that I get.

Reply:
The location of the chart have very recently changed. Perhaps it's worth while to make sure nothing has been lost in the transition?

Reply:
@effi-ofer : this file was already in git. You did good to `make chart`. You have repaired the inconsistency that I called out above.